src/workload.rs

```diff
@@ -537,7 +553,12 @@ mod tests {
             name: "".to_string(),
             namespace: "".to_string(),
             service_account: "".to_string(),
+            workload_name: "".to_string(),
```

Comment:
We should either rely on `default::default()` more or have a test module with some example workloads to re-use in tests or both.